### PR TITLE
morph/client: Return complete eACL signature from contract

### PR DESCRIPTION
**Pull request by alexvanin**
Related to https://github.com/nspcc-dev/neofs-contract/pull/35

### CODE REVIEW

File: cmd/neofs-cli/modules/container.go [hunk 1]
```diff
@@ -425,10 +425,10 @@ Container ID in EACL table will be substituted with ID from the CLI.`,
 			for i := 0; i < awaitTimeout; i++ {
 				time.Sleep(1 * time.Second)
 
-				eaclSig, err := cli.GetEACLWithSignature(ctx, id, globalCallOptions()...)
+				table, err := cli.GetEACL(ctx, id, globalCallOptions()...)
```
Comment (cthulhu-rider) [inline cmd/neofs-cli/modules/container.go:428]: I suggest to leave usage of `GetEACLWithSignature` method, but with signature check.

Comment (alexvanin) [reply cmd/neofs-cli/modules/container.go:428]: `GetEACL` method encapsulates signature check inside. I don't think explicit signature check is an important thing there.
We have `GetEACLWithSignature` in `neofs-cli get-eacl` command if you want to provide use case example for this function.

Comment (cthulhu-rider) [reply cmd/neofs-cli/modules/container.go:428]: If we use GetEACLWithSignature, we still can output the signature event if it is wrong. But this is not necessary.

**Review by cthulhu-rider — APPROVED** [review]

File: cmd/neofs-cli/modules/container.go (hunk 1 above)
Comment (alexvanin) [reply cmd/neofs-cli/modules/container.go:428]: > If we use GetEACLWithSignature, we still can output the signature event if it is wrong. But this is not necessary.

That is exactly how it's done in `get-eacl` command. As for `--await` flag in `set-eacl` I don't think it is necessary there.

**Pull request merged**